File: client-app/src/features/AppNotifications/AppNotifications.ts
import { isNullishCoalesce } from "typescript";
import { AppNotificationsMaster, IAppNotificationsMaster } from "../AppNotificationsMaster/AppNotificationsMaster";

export interface IAppNotifications {
	Id: number
	NotificationsMasterId: number
	UserId: string
	NextReminderTime: Date
	NoOfReminders: number
	ReadStatus: boolean
	NotificationsMaster : IAppNotificationsMaster
}

export class AppNotifications implements IAppNotifications {
	Id: number = 0;
	NotificationsMasterId: number = 0;
	UserId: string = '';
	NextReminderTime: Date = new Date();
	NoOfReminders: number = 0;
	ReadStatus: boolean = false;
	NotificationsMaster : IAppNotificationsMaster = new AppNotificationsMaster();
  
  constructor(init?: IAppNotifications) {
    Object.assign(this, init);
  }
}

File: Domain/AppNotifications.cs
using System;

namespace Domain
{
    public class AppNotifications
    {
		public int Id { get; set; }
		public int NotificationsMasterId { get; set; }
		public string UserId { get; set; }
		public bool ReadStatus { get; set; }
		 public virtual AppNotificationsMaster NotificationsMaster { get; set; }
		// public DateTime NextReminderTime { get; set; }
		// public int NoOfReminders { get; set; }
		
    }
}

















